pkg/etcdcli/health.go

```diff
@@ -41,6 +41,10 @@ type healthCheck struct {
 
 type memberHealth []healthCheck
 
+func GetMemberHealth(ctx context.Context, etcdMembers []*etcdserverpb.Member) memberHealth {
```

Comment:
this seems duplicated, why not make the below method public?

Reply:
on it 👍🏽